### ISSUE TITLE
How to enable nvlink P2P across container in the same node?

### ISSUE THREAD

**Issue by autumn0207** [NVIDIA/nccl#945]
Has anyone have the same problem？ only SHM can work across container in same node

**Comment by KaimingOuyang** [comment]
I don't think you can do that. Since each container is a separate "node", cuda driver cannot detect any chances for P2P.

**Comment by autumn0207** [comment]
@KaimingOuyang maybe we can hijack cuda api in container and response the result in host

**Comment by sjeaugey** [comment]
We're not aware of any such possiblity.

**Comment by stephenmsachs** [comment]
### Issue Cleanup: Helping Us Focus on Current Challenges

  We're [reviewing](https://github.com/NVIDIA/nccl/discussions/1761) older issues to ensure we prioritize the most relevant and active ones. Since this issue hasn't seen updates in over 6 months, we'll be closing it for now.

  ,*This change helps us focus our efforts on addressing any current issues our users are facing.* If this issue still affects you, please don't hesitate to reopen it with a quick update (e.g., "Still relevant on [version=X]").
  Thanks for your understanding and for contributing to NCCL.
